WordPress/gutenberg
# Mobile: Remove size settings

## Description

This is a small PR to remove the option to edit sizes in Image Blocks.

I also took the opportunity to organize import statements in `image/edit.native.js`.

<img width="338" alt="screen shot 2019-02-07 at 9 58 20 pm" src="https://user-images.githubusercontent.com/9772967/52442479-82cf8a80-2b23-11e9-9bc6-ba7fd6a44f1a.png">

**To test:**
- Check out the [related `gutenberg-mobile` branch](https://github.com/wordpress-mobile/gutenberg-mobile/pull/570) and run the app.
- Add an image block and add an image to it.
- Open the image settings.
- Check that the Image Size option is **not** there.
